python/semantic_kernel/connectors/ai/google/vertex_ai/vertex_ai_prompt_execution_settings.py

```diff
@@ -35,19 +36,21 @@ class VertexAITextPromptExecutionSettings(VertexAIPromptExecutionSettings):
 class VertexAIChatPromptExecutionSettings(VertexAIPromptExecutionSettings):
     """Vertex AI Chat Prompt Execution Settings."""
 
-    tools: list[dict[str, Any]] | None = Field(None, max_length=64)
-    tool_choice: str | None = None
+    # Do not set the tools and tool_config manually.
+    # They are set by the service based on the function choice configuration.
+    tools: list[Tool] | None = Field(None, max_length=64)
```

Comment:
I think we can use the Pydantic Field description to include these comments that they shouldn't be set manually? Thoughts?